Decent.Minecraft.Client/Java/JavaBlock.cs

```diff
@@ -80,7 +80,9 @@ static JavaBlock()
                 d == 1 ? new MossyStoneBricks() :
                 d == 2 ? new CrackedStoneBricks() :
                 (StoneBricks)new ChiseledStoneBricks();
-            _ctors[Id<Wood>()] = d => new Wood((WoodSpecies)(d & 0x3), (Axis)(d & 0xC));
+            // Wood is both 17 and 162 depending on species
+            _ctors[17] = d => new Wood((WoodSpecies)(d & 0x3), (Axis)(d & 0xC));
```

Comment:
Oh yes, I'm very much in agreement. That's what I wanted all along ;) That sort of thing is all over: it really shows how MC was grown organically. Our class hierarchy should be more rational than that.

On snow, the behavior is one thing, thickness is another. I'll check later, but from what I remember, the wiki spec and reality were out of sync, and the value 8 wasn't actually possible except through a snow block.